src/modules/account/components/TransactionComponent.vue

```diff
@@ -20,9 +20,9 @@
         <Column style="width: 200px" v-if="!wideScreen">
           <template #body="{data}">
             <div>
-              <div class="uppercase text-xs text-gray-300 font-bold mb-1 break-all">Tx Hash</div>
+              <div class="uppercase text-xs text-gray-300 font-bold mb-1">Tx Hash</div>
               <router-link class="uppercase font-bold text-xs block" :to="{ name: 'ViewTransaction', params:{ hash: data.hash }}">
-                <span class="text-xs break-all text-blue-600 hover:text-blue-primary hover:underline" v-tooltip.right="data.hash">{{ data.hash.substring(0, 15) }}..</span>
+                <span class="text-blue-600 hover:text-blue-primary hover:underline inline-flex text-ellipsis overflow-hidden w-44" v-tooltip.right="data.hash">{{ data.hash }}</span>...
```

Comment:
Hi @haieng1234 , I think if the full txn is displayed by default. we can remove tooltip

Reply:
Will make the change accordingly thanks